docker-compose.yml

```diff
@@ -18,7 +28,16 @@ services:
       - rabbitmq
     networks:
       - mynetwork
-
+  
+  srv_consumer:
+    build: ./srv_consumer
```

Comment:
srv_consumer works but crashes and exit when can't connect to rabbit (ex. srv_consumer starts before rabbitmq) or mongo.
If you don't want to handle these failures as srv_publisher you could add `restart: always` here.
https://docs.docker.com/config/containers/start-containers-automatically/

Reply:
Very nice catch! I was struggling to figure out why I couldn't connect to rabbit and that was the reason. Thanks